pkg/cmd/pipelineresource/create.go

```diff
@@ -284,10 +284,19 @@ func (res *resource) askClusterParams() error {
 		res.pipelineResource.Spec.Params = append(res.pipelineResource.Spec.Params, usernameParam)
 	}
 
-	insecureParam, err := askParam("insecure", res.askOpts)
+	insecureAns, err := askToSelect("Is the cluster secure?", []string{"true", "false"}, res.askOpts)
```

Comment:
what is this about ?

and if we are asking for a confirmation can we use the standard yes no?